src/test/regress/expected/local_shard_execution.out

```diff
@@ -2056,7 +2056,7 @@ NOTICE:  executing the command locally: UPDATE public.event_responses_1480001 ev
 SELECT count(*) FROM event_responses WHERE event_id = 16;
 DEBUG:  Distributed planning for a fast-path router query
 DEBUG:  Creating router plan
-DEBUG:  Plan is router executable
+DEBUG:  query has a single distribution column value: 16
```

Comment:
We're now printing the dist key for fast-path router queries as well, which is an improvement